ios/NewExpensify/RCTBootSplash.mm

```diff
@@ -145,34 +149,36 @@ - (NSDictionary *)constantsToExport {
   };
 }
 
-- (void)hideImpl:(BOOL)fade
-         resolve:(RCTPromiseResolveBlock)resolve {
-    if (_resolveQueue == nil)
-      _resolveQueue = [[NSMutableArray alloc] init];
++ (void)bringSubviewToFrontIfInitialized {
+  if(![self isInitialized]) {
+    return;
+  }
 
-    [_resolveQueue addObject:resolve];
+  [_rootView bringSubviewToFront:_loadingView];
```

Comment:
What will happen if `_loadingView` is not a subview of `_rootView`? Or is it not possible?

Reply:
It has been moved from OD because we no longer have access to those values there and the logic remains unchanged

<img width="678" alt="image" src="https://github.com/user-attachments/assets/9b819551-4906-44ff-bf57-a01673620a2e" />